### PR TITLE
WRO-7313: Update sample's index.js with the new selective hydrateRoot pattern

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated package.json with latest rc.1 version
Updated sample's index.js with hydrateRoot pattern

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-7313

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

File: samples/pattern-video-player/src/index.js [hunk 1]
```diff
@@ -1,14 +1,17 @@
-import {createRoot} from 'react-dom/client';
+/* global ENACT_PACK_ISOMORPHIC */
+
```
Comment (MikyungKim) [inline samples/pattern-video-player/src/index.js]: Let's remove the line.

File: samples/qa-fonts/src/components/FontList/FontList.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ const FontList = kind({
 					return (
 						<SlotItem key={'item' + i}>
 							<slotBefore>
-								{document.fonts.check(font) ?
+								{typeof document !== 'undefined' && document.fonts.check(font) ?
```
Comment (MikyungKim) [inline samples/qa-fonts/src/components/FontList/FontList.js]:
```suggestion
								{typeof window !== 'undefined' && document.fonts.check(font) ?
```

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**